charms/istio-pilot/src/resources_handler.py

```diff
@@ -0,0 +1,103 @@
+#! /usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Resources handling library using Lightkube."""
+
+import yaml
+from jinja2 import Environment, FileSystemLoader
+from lightkube import Client, codecs
+from lightkube.core.exceptions import ApiError
+from lightkube.generic_resource import create_namespaced_resource, GenericNamespacedResource
+
+
+class ResourceHandler:
+    def __init__(self, config):
```

Comment:
I left it because originally I was receiving more than just `app_name` and `model_name`,things such as the context for rendering manifests, paths, etc. But probably I can simplify this, as you said, just receiving those two. I'll change it.